src/components/Home/HomeContent.tsx

```diff
@@ -64,11 +60,11 @@ const HomeContent = ({ personalities, href, title, debateClaims, reviews }) => {
                     />
                 </Grid>
 
-                {!isLoggedIn && (
-                    <Grid item xs={12} lg={9} order={3}>
-                        <CTARegistration />
-                    </Grid>
-                )}
+
+                <Grid item xs={12} lg={9} order={3}>
+                    <CTARegistration />
```

Comment:
Blocking: This is still rendering the old CTARegistration component. Can you update both the import and usage here to the newly renamed CTA component